virtual_rainforest/core/constants_loader.py

```diff
@@ -108,14 +66,25 @@ def load_constants(config: Config, model_name: str, class_name: str) -> Any:
     """
 
     # Extract dataclass of interest from registry
-    constants_class = CONSTANTS_REGISTRY[model_name][class_name]
-
-    # Check if any constants have been supplied
-    if model_name in config and "constants" in config[model_name]:
+    try:
+        constants_class = MODULE_REGISTRY[model_name]["constants"][class_name]
+    except KeyError as excep:
+        LOGGER.critical(f"Unknown constants class: {model_name}.{class_name}")
+        raise excep
+
+    # Check if any constants have been supplied for this constants class
```

Comment:
Is it possible to create a constants class which doesn't supply constants?

Reply:
Badly phrased comment - the config might not supply values, in which case defaults are used.